src/ripple/basics/hardened_hash.h

```diff
@@ -159,13 +148,8 @@ class basic_hardened_hash<HashAlgorithm, false>
     template parameter (the hashing algorithm).  For details
     see https://131002.net/siphash/#at
 */
-#if RIPPLE_NO_HARDENED_HASH_INSTANCE_SEED
-template <class HashAlgorithm = beast::xxhasher>
-    using hardened_hash = basic_hardened_hash<HashAlgorithm, true>;
```

Comment:
Since this is the only time `true` is used, I think you can get rid of the `bool ProcessSeeded` template parameter and the `true` overload entirely.

Reply:
I seriously thought about doing that, and will if that's what the team wants.  But I didn't because of what I wrote in the PR description above:

> I've left in the (currently unused) code to seed hardened_hash on a per-process basis so that we can easily choose to do so on a per-container basis should the need arise.

Reply:
I should have addressed that. My concern is more about bit-rot. To me, it's almost like leaving code commented out in case we need it later. I figure that if we delete it, and there is a need down the road, worst case, we can dig through git history and resurrect it.

But I don't feel very strongly about it, and if the consensus is to leave it, I'm not going to die on that hill.

Reply:
Me neither.  Thanks for the comment, and let's see what others say.  I'm happy to go with the consensus.

Reply:
As a somewhat interested observer, I'm weakly in favor of removing code that is unused.

Reply:
Two votes for removal is actionable for me.  I'll push a [FOLD] in the morning.  Thanks for the input.

Reply:
Pushed the removal.